wolfcrypt/src/asn.c

```diff
@@ -22487,57 +22449,59 @@ static int EncodePublicKey(int keyType, byte* output, int outLen,
  * All extensions supported for encoding are described.
  */
 static const ASNItem certExtsASN[] = {
+            /* Basic Constraints Extension - 4.2.1.9 */
```

Comment:
Also here, the CSR request extensions had an extra level of sequence that was invalid. Just for ASN template.